src/pages/staking-hub/onboarding/step0/1createSafe.tsx

```diff
@@ -117,23 +117,25 @@ const CreateSafe = () => {
           </IconButton>
         </FlexContainer>
       </Container>
-      {owners.length > 1 &&
-        owners.slice(1).map((owner) => {
-          return (
-            <FlexContainer key={owner.id}>
-              <TextField
-                label="Address"
-                placeholder="New owner address here..."
-                fullWidth
-                onChange={(e) => updateOwnerAddress(owner.id, e.target.value)}
-              />
-              <IconButton onClick={() => removeOwner(owner.id)}>
-                <DeleteIcon />
-              </IconButton>
-            </FlexContainer>
-          );
-        })}
-
+      <div>
+        {owners.length > 1
+          ? owners.slice(1).map((owner) => {
+            return (
+              <FlexContainer key={owner.id}>
+                <TextField
+                  label="Address"
+                  placeholder="New owner address here..."
+                  fullWidth
+                  onChange={(e) => updateOwnerAddress(owner.id, e.target.value)}
+                />
+                <IconButton onClick={() => removeOwner(owner.id)}>
+                  <DeleteIcon />
+                </IconButton>
+              </FlexContainer>
+            );
+          })
+          : null}
+      </div>
```

Comment:
I do not see the reason why you would change the `a && b` into  `a ? b  :  null` here and leave the  `a && b`  logic below?

Reply:
yeah this could stay how it was

Reply:
while on it, you can also bring back `{(is_xDAI_to_safe_loading || is_wxHOPR_to_safe_loading) && <span>Check your Wallet...</span>}`

Reply:
brought all of that back